### DISCUSSION TITLE
Conversion error

### DISCUSSION THREAD

**Discussion by cdefghijkl** [TheLastBen/fast-stable-diffusion#1170] [General]
![Screenshot 2022-12-27 at 17-48-57 Google Colaboratory](https://user-images.githubusercontent.com/121331850/209656927-f8e51774-d6ec-404f-baa1-91647fe9dbd9.png)

When I model download from this link it show this error: 
https://huggingface.co/cdefghijkl/lmx/tree/main

Not only that link, but other ckpt who has no other additional files in their repo.

**Comment by cdefghijkl** [comment]
up

**Comment by cdefghijkl** [comment]
up

**Comment by TheLastBen** [comment]
I tried it, couldn't convert, you need to find the diffusers version of the model, try contacting the creator of the model

**Reply by cdefghijkl** [reply]
The creator is confused because the model is from merging model with another model so he has no diffuser version. Fortunately, there's a HG space which can convert ckpt to diffuser, at first it not works because of a bug but now it already fixed.

**Reply by jbrsfa** [reply]
> The creator is confused because the model is from merging model with another model so he has no diffuser version. Fortunately, there's a HG space which can convert ckpt to diffuser, at first it not works because of a bug but now it already fixed.

could you share a link?

**Reply by cdefghijkl** [reply]
https://huggingface.co/spaces/camenduru/converter

**Reply by jbrsfa** [reply]
> https://huggingface.co/spaces/camenduru/converter

which conversion option should I choose to convert ckpt to diffuser?

**Reply by cdefghijkl** [reply]
top one. ckpt to pytorch

**Reply by KodyAG** [reply]
> top one. ckpt to pytorch

I just get an error, how does this work?

**Reply by cdefghijkl** [reply]
First click all of delete buttons there and click download ckpt then convert to pt then push pt to hg

**Reply by cdefghijkl** [reply]
I tried again and it seems there's a problem for now.

**Reply by omegaelite** [reply]
still has a problem for me as well.

**Reply by cdefghijkl** [reply]
It's working now.
